.github/workflows/build-dependabot.yml

```diff
@@ -1,4 +1,4 @@
-name: Build project for Dependabot
```

Comment:
Maybe its better fo have different title as Build.

Reply:
Added the all repository changes list